app/scripts/IngresseAPI.ErrorInterceptor.js

```diff
@@ -213,6 +213,11 @@ angular.module('ingresseSDK')
               return $q.reject(error);
             }
 
+            if (error.code === 3030) {
+              error.message = 'Não encontramos este e-mail em nossa plataforma. Certifique-se de que ele foi utilizado para criar uma conta na Ingresse.';
```

Comment:
i think we should remove the `Ingresse` from this message because this error will show in others whitelables

Reply:
done